src/widgets/pictures/ui/pictures.scss

```diff
@@ -7,24 +7,42 @@
   .carousel-slider {
     .control-next,
     .control-prev {
-      top: calc(50% - 50px);
+      --slide-arrow-btn-height: 45px;
 
-      height: 100px;
+      top: calc(50% - var(--slide-arrow-btn-height) / 2);
+
+      height: var(--slide-arrow-btn-height);
+      margin: 2px;
 
       opacity: $opacity-100;
       backdrop-filter: blur(20px);
       border-radius: 8px;
+
+      &:hover {
```

Comment:
Maybe we need to use here @media (hover: hover)? Because if you use a touch screen the white color on the background will stick.